src/main/java/org/myrobotlab/net/BareBonesBrowserLaunch.java

```diff
@@ -29,7 +31,13 @@
   static final String errMsg = "Error attempting to launch web browser";
 
   public static void openURL(String url) {
-    try { // attempt to use Desktop library from JDK 1.6+
+    try { 
+      
+      if (!GraphicsEnvironment.isHeadless()) {
+        log.warn("graphical browser requested but currently no display");

```

Comment:
Is this logic backwards?  And if the env is headless, wouldn't you want to just return and not spawn the browser?